cekit/cli.py

```diff
@@ -1,313 +1,345 @@
 # -*- coding: utf-8 -*-
 
-import argparse
-import os
 import logging
+import os
+import shutil
 import sys
 
+import click
 
 from cekit.config import Config
-from cekit.builder import Builder
-from cekit.log import setup_logging
 from cekit.errors import CekitError
-from cekit.generator.base import Generator
-from cekit.test.collector import TestCollector
-from cekit.test.runner import TestRunner
-from cekit.tools import cleanup, DependencyHandler
+from cekit.log import setup_logging
+from cekit.tools import Map
 from cekit.version import version
 
-# FIXME we shoudl try to move this to json
+# FIXME we should try to move this to json
 setup_logging()
-logger = logging.getLogger('cekit')
-config = Config()
+LOGGER = logging.getLogger('cekit')
+CONFIG = Config()
```

Comment:
Is there a reason why in some code its 'logger' and some its 'LOGGER' ? Could we have it all lower case?

Reply:
It should be upper case everywhere by Python convention. I haven't updated it everywhere yet.

Reply:
https://www.python.org/dev/peps/pep-0008/#constants

Reply:
What are you doing replying at this time :-) I'e just realised its PEP8 having looked it up , sorry for the noise ;-)
Yeah, probably needs to be updated everywhere

Reply:
I could ask you the same thing :P